pandas/core/indexes/multi.py

```diff
@@ -209,7 +209,7 @@ class MultiIndex(Index):
 
     def __new__(cls, levels=None, labels=None, sortorder=None, names=None,
                 copy=False, verify_integrity=True, _set_identity=True,
```

Comment:
can you put dtype before copy, makes this signature close to what we do elsewhere in constructors

Reply:
Actually not elsewhere :) but ok, it's not a big deal
https://github.com/pandas-dev/pandas/blob/fd010de51bd21e4240daac2e1fd0044c65a45091/pandas/core/indexes/datetimes.py#L329-L333

Reply:
> Actually not elsewhere :) but ok, it's not a big deal

@PoppyBagel is it consistent elsewhere?  Having them be consistent would be nice.